src/java.base/share/classes/jdk/internal/reflect/NativeConstructorAccessorImpl.java

```diff
@@ -28,14 +28,20 @@
 import sun.reflect.misc.ReflectUtil;
 
 import java.lang.reflect.*;
+import jdk.internal.misc.Unsafe;
 
 /** Used only for the first few invocations of a Constructor;
     afterward, switches to bytecode-based implementation */
 
 class NativeConstructorAccessorImpl extends ConstructorAccessorImpl {
+    private static final Unsafe unsafe = Unsafe.getUnsafe();
+    private static final long accessorGeneratedaOffset
```

Comment:
Typo "Generated*a*Offset". But also this is `static final`, so it should be e.g. `GENERATED_OFFSET`. The field can be just `generated` to match the name of this offset.

Reply:
field name udpated